tests/specs/analog/uplinkCommands.spec.js

```diff
@@ -0,0 +1,58 @@
+import {test, expect} from '@playwright/test';
+import {MainPage} from '../../objects/MainPage.js';
+import fixture from '../../fixtures/main.js';
+import {uplinkSimpleCommands} from '../../fixtures/analog/uplinkCommands.js';
+
+
+const checkingParsedMessage = async ( page, format, command ) => {
```

Comment:
```suggestion
const checkCommand = async ( page, format, command ) => {
```
looks better, according to usage in `for ( const command in uplinkSimpleCommands ) {...`

Reply:
rename to validateUplinkCommands